### ISSUE TITLE
PLS add Letters

### ISSUE THREAD

**Issue by Bruhh1234** [Invy55/SimpleBanner#6]
Quick question, cann you add Letter banner´s ? like:  
ABCDEFGHIJKLMNOPQRSTUVWXYZ 1234567890

LIke this PL
Thx

**Comment by Invy55** [comment]
> Quick question, cann you add Letter banner´s ? like:
> ABCDEFGHIJKLMNOPQRSTUVWXYZ 1234567890
> 
> LIke this PL
> 

Is it a new feature of minecraft or you are asking for something that adds them automatically using different patterns?

**Comment by Bruhh1234** [comment]
no it is not a new feature, but would like to have letter banners xD

**Comment by Bruhh1234** [comment]
you think you can add ?

**Comment by Invy55** [comment]
> you think you can add ?

yes of course

**Comment by Bruhh1234** [comment]
Very Very Nice, can You let me Know when you have add ?

Best THX

**Comment by Bruhh1234** [comment]
short quesstion : Have you add ?
Best Gretzz

**Comment by Invy55** [comment]
> short quesstion : Have you add ?
> Best Gretzz

Yep, but poggit seem not publishing it :/

**Comment by Invy55** [comment]
> short quesstion : Have you add ?
> Best Gretzz

ah no sorry don't have quite lot of time if u want u can help forking and them i'll merge it? else when i'll have time i'll make

**Comment by Bruhh1234** [comment]
yes i can, just give me an Example how i can doo this in your code 

THX

**Comment by Invy55** [comment]
> yes i can, just give me an Example how i can doo this in your code
> 
> THX

You know what, do u have any website where I can see how to make the letters?

**Comment by Bruhh1234** [comment]
have found this video 
https://www.bing.com/videos/search?q=minecraft+buchstaben+banner&&view=detail&mid=9F0D3BC4791414AB7E179F0D3BC4791414AB7E17&rvsmid=8D31F34A1E7B33B87C128D31F34A1E7B33B87C12&FORM=VDRVRV

**Comment by Bruhh1234** [comment]
https://auscraft.net/forums/threads/banner-alphabet.2848/

**Comment by Invy55** [comment]
> https://auscraft.net/forums/threads/banner-alphabet.2848/

Cool I wanted to add a guy to make everyone create their patters and putting this in the defaults, but i don't have quite a lot time so i'll try in my free time

**Comment by Bruhh1234** [comment]
yea, you can give me an Example For Letter A an i make the rest ?

**Comment by Bruhh1234** [comment]
so can i help ?

**Comment by Invy55** [comment]
> so can i help ?

I don't think, I have to implement a new logic then you can create the patterns for me

**Comment by Bruhh1234** [comment]
Okay let me know when you ready, have time for this This evening, 
do you think you have the new logic ready by then, then i could finish the letters tonight by night
Best Gretzz

**Comment by Invy55** [comment]
> Okay let me know when you ready, have time for this This evening,
> do you think you have the new logic ready by then, then i could finish the letters tonight by night
> Best Gretzz

I don't think today I can do anything

**Comment by galaxytwenty** [comment]
i too need letterbanner´s, have you made this?

**Comment by Invy55** [comment]
no sorry, im not quite active and i can't find time to

**Comment by galaxytwenty** [comment]
can jou make just one letterbanner like "A", Then i Make the rest of it Like "B""C""D" etc...

this was rly nice
